ledger/ledger-api-test-tool/src/main/scala/com/daml/ledger/api/testtool/Tests.scala

```diff
@@ -75,24 +71,27 @@ object Tests {
         val latencyKey: String = performanceEnvelopeLatencyTestKey(envelope)
         val transactionSizeKey: String = performanceEnvelopeTransactionSizeTestKey(envelope)
         List(
-          throughputKey -> (new testtool.tests.PerformanceEnvelope.ThroughputTest(
+          new testtool.tests.PerformanceEnvelope.ThroughputTest(
+            throughputKey,
             logger = LoggerFactory.getLogger(throughputKey),
             envelope = envelope,
             reporter = reporter,
-          )(_)),
-          latencyKey -> (new testtool.tests.PerformanceEnvelope.LatencyTest(
+          ),
+          new testtool.tests.PerformanceEnvelope.LatencyTest(
+            latencyKey,
             logger = LoggerFactory.getLogger(latencyKey),
             envelope = envelope,
             reporter = reporter,
-          )(_)),
-          transactionSizeKey -> (new testtool.tests.PerformanceEnvelope.TransactionSizeScaleTest(
+          ),
+          new testtool.tests.PerformanceEnvelope.TransactionSizeScaleTest(
+            transactionSizeKey,
             logger = LoggerFactory.getLogger(transactionSizeKey),
             envelope = envelope,
-          )(_)),
+          ),
         )
       }
     }
-  }.toMap
+  }.toSeq
```

Comment:
```suggestion
  }
```
Looks like `Envelope.values` is already a `List`, is `toSeq` really needed? `List` is a `Seq`.